tests/test__collection_api.py

```diff
@@ -1406,3 +1406,87 @@ def test_aggregate_unwind_push_first(self):
                 '_id': 'ooo'
             }]
         self.assertEqual(expect, list(actual))
+
+    def test_aggregate_unwind_pushall(self):
```

Comment:
Please try to compact this code, it unnecessarily spans across a lot of lines...